src/App.css

```
@@ -1,6 +1,58 @@
+/* @import "tailwindcss"; */
 #root {
   max-width: 1280px;
   margin: 0 auto;
   padding: 2rem;
   text-align: center;
+}
+
+body {
+  background: #0f172a;
+  color: #e2e8f0;
+  font-family: "Inter", sans-serif;
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)